tests/test_task_runners.py

```diff
@@ -32,3 +34,55 @@ class TestConcurrentTaskRunner(TaskRunnerStandardTestSuite):
     @pytest.fixture
     def task_runner(self):
         yield ConcurrentTaskRunner()
+
+class TestConcurrentTaskRunnerLimitedThreads(TaskRunnerStandardTestSuite):
+    @pytest.fixture
+    def task_runner(self):
+        yield ConcurrentTaskRunner(max_workers=3)
+
+class TestConcurrentTaskRunnerSingleThreaded(TaskRunnerStandardTestSuite):
+    @pytest.fixture
+    def task_runner(self):
+        yield ConcurrentTaskRunner(max_workers=1)
```

Comment:
You may need to / be able to change the the `concurrency_type` on this instance of the task runner from `TaskConcurrencyType.CONCURRENT` to `TaskConcurrencyType.SEQUENTIAL` so it goes through the sequential standard tests